editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -1447,55 +1464,25 @@ export var storyboard = (
 )
 `)
       })
-      it('cannot copy element that has code in its children array', async () => {
-        const editor = await renderTestEditorWithCode(
-          `import * as React from 'react'
-          import { Storyboard } from 'utopia-api'
-          
-          const width = 122
-          
-          export var storyboard = (
-            <Storyboard data-uid='sb'>
-              <div
-                style={{
-                  backgroundColor: '#aaaaaa33',
-                  position: 'absolute',
-                  left: 281,
-                  top: 329,
-                  width: 122,
-                  height: 73,
-                }}
-                data-uid='container'
-              >
-                <div
-                  style={{
-                    backgroundColor: '#aaaaaa33',
-                    position: 'absolute',
-                    left: 19,
-                    top: 19,
-                    width: width,
-                    height: 40,
-                  }}
-                  data-uid='child'
-                />
-              </div>
-            </Storyboard>
-          )
-          `,
-          'await-first-dom-report',
-        )
+      describe('repeated paste', () => {
+        async function pasteNTimes(editor: EditorRenderResult, n: number) {
+          const canvasRoot = editor.renderedDOM.getByTestId('canvas-root')
 
-        await selectComponentsForTest(editor, [EP.fromString('sb/container')])
+          for (const _ in Array(n).fill(0)) {
+            // paste 4 times
+            firePasteEvent(canvasRoot)
 
-        await expectNoAction(editor, () => pressKey('c', { modifiers: cmdModifier }))
-        await editor.getDispatchFollowUpActionsFinished()
+            // Wait for the next frame
+            await clipboardMock.pasteDone
+            await editor.getDispatchFollowUpActionsFinished()
+
+            await pressKey('l')
```

Comment:
`'l'`?

Reply:
I chose a key that wasn't tied to an existing shortcut, actually is indicates the problem that we agreed to fix before the FS is removed from this feature, so this should be changed to Esc or any other non-digit or tab key in the long run